docs/index.md

```diff
@@ -116,6 +117,30 @@ Passing a token to the verification endpoint will return a 200 response and the
 $ curl -X POST -H "Content-Type: application/json" -d '{"token":"<EXISTING_TOKEN>"}' http://localhost:8000/api-token-verify/
 ```
 
+## Blacklist Token
+If `JWT_ENABLE_BLACKLIST` is True, and `rest_framework_jwt.blacklist` has been added to `INSTALLED_APPS`, tokens can be made invalid (prior to expiration) by blacklisting them. More information on the concept of JTI (JWT ID) and blacklisting tokens can be read [here](http://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#jtiDef) and [here](https://auth0.com/blog/2015/03/10/blacklist-json-web-token-api-keys/).
+
+This package comes with a default implementation that stores the blacklisted tokens in the configured django database and includes an admin integration.
+
+To use this feature, add a URL like so:
+
+```python
+    url(r'^api-token-blacklist/', 'rest_framework_jwt.views.blacklist_jwt_token'),
```

Comment:
does this doc need to be updated to point to the view as belonging to the blacklist app?